package.json

```diff
@@ -7,9 +7,12 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.1.3",
+    "bootstrap": "^5.2.2",
```

Comment:
This will be of great use to us 👍

Reply:
This is a code review, no need to cluter the review board with this.